server.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express")
+const app = express();
+
+
+app.listen(3000,()=>{
```

Comment:
Consider adding error handling for the server. This can be done by adding a middleware function after all other `app.use()` and route calls. This function would take four arguments: `(err, req, res, next)`. Inside this function, you can handle errors as you see fit, such as logging them and sending a response to the client. [important]